doc/release/legacy-policy.rst

```diff
@@ -54,7 +56,7 @@ release series:
 
 *   **LTS (Long Term Support)** is a release series that is supported
     for 3 years (community) and up to 5 years (paying customers).
-    Current LTS release series is 1.10, and it receives only PATCH level
+    Current LTS release series is ``1.10``, and it receives only PATCH level
```

Comment:
No. Current LTS is 2.11. The 1.10 series is EOL and EOS - no more patches expected.